File: app/src/main/java/com/BlizzardArmory/diablo/item/Primary.java
package com.BlizzardArmory.diablo.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class Primary {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("textHtml")
    @Expose
    private String textHtml;

    public String getText() {
        return text;
    }

    public String getTextHtml() {
        return textHtml;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/diablo3/item/Primary.kt
package com.BlizzardArmory.model.diablo.diablo3.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Primary.
 */
@Keep
data class Primary(

    @SerializedName("text")

    val text: String,


    @SerializedName("textHtml")

    val textHtml: String

)